src/lib/fauna-account-client.mjs

```diff
@@ -21,6 +43,18 @@ export class FaunaAccountClient {
     return dashboardOAuthURL;
   }
 
+  /**
+   * Retrieves an access token from the Fauna account API.
+   *
+   * @param {Object} opts - The options for the token request.
+   * @param {string} opts.clientId - The client ID for the OAuth application.
+   * @param {string} opts.clientSecret - The client secret for the OAuth application.
+   * @param {string} opts.authCode - The authorization code received from the OAuth authorization.
+   * @param {string} opts.redirectURI - The redirect URI for the OAuth application.
+   * @param {string} opts.codeVerifier - The code verifier for the OAuth PKCE flow.
```

Comment:
are any of these optional? if so, please use the JSDoc optional annotation, e.g.:
```
@param {string} [opts.codeVerifier] - The code verifier for the OAuth PKCE flow.
```

if not, disregard